### PR TITLE
New Workflow Changes

### PR DESCRIPTION
**Pull request by lispandfound**
Small changes for the new workflow:

1. Add a coordinate patchgrid function to `qcore.grid`. This function is used to generate geometry for SRFs. The difference between this and  a coordinate grid is easily summarised in the following image:
image
2. Add the mag scaling packages to `setup.py`. This made them easier to use in workflow and source modelling

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 setup(
     name="qcore",
     version="1.2",
-    packages=["qcore"],
+    packages=["qcore", "qcore.uncertainties", "qcore.uncertainties.magnitude_scaling"],
```
Comment (joelridden) [inline setup.py:8]: make a __init__.py instead for these

**Review by joelridden — COMMENTED** [review]
We should at some point write tests for grid.py?

**Comment by sungeunbae** [comment]
> Small changes for the new workflow:
> 
> 1. Add a coordinate patchgrid function to `qcore.grid`. This function is used to generate geometry for SRFs. The difference between this and  a coordinate grid is easily summarised in the following image:
>    image
 
 Meaning a grid made of centroids?

**Review by sungeunbae — DISMISSED** [review]

**Review by joelridden — DISMISSED** [review]

**Comment by lispandfound** [comment]
> We should at some point write tests for grid.py?

This is a good idea. I'll include some now.

**Review by joelridden — APPROVED** [review]

**Review by AndrewRidden-Harper — APPROVED** [review]

**Pull request merged**